devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -12,15 +12,19 @@
  *
  */
 metadata {
-	definition (name: "Z-Wave Dimmer Switch Generic", namespace: "smartthings", author: "SmartThings") {
+	definition (name: "Z-Wave Dimmer Switch Generic", namespace: "smartthings", author: "SmartThings", category: "C5") {
 		capability "Switch Level"
 		capability "Actuator"
 		capability "Switch"
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
+		capability "Configuration"
+		capability "Health Check"
 
 		fingerprint inClusters: "0x26", deviceJoinName: "Z-Wave Dimmer"
+		fingerprint mfr:"0063", prod:"4944", deviceJoinName: "Z-Wave In-Wall Dimmer"
```

Comment:
@jackchi @juano2310 
When we pair GE12724, the device type shown in staging IDE is **Z-wave Dimmer Switch Generic**.
So which DTH will the device refer to when paired?
Though I've made the changes in **dimmer-switch.groovy**, will the device refer to it?
As the _in-clusters_ is matching in **zwave-dimmer-switch-generic.groovy**
and the _mfr_ & _prod_ is matching in **dimmer-switch.groovy**